website/src/parsers/yaml/yaml-unist-parser.js

```diff
@@ -0,0 +1,35 @@
+import defaultParserInterface from '../utils/defaultParserInterface';
+import pkg from 'yaml-unist-parser/package.json';
+
+const ID = 'yaml-unist-parser';
+
+export default {
+  ...defaultParserInterface,
+
+  id: ID,
+  displayName: ID,
+  version: pkg.version,
+  homepage: pkg.homepage || 'https://www.npmjs.com/package/yaml-unist-parser',
+
+  _ignoredProperties: new Set(['parent', 'errors', 'type']),
```

Comment:
There's no `errors` prop and the `parent` is defined as non-enumerable prop (internal usage) so I guess we don't need to specify it here?

Reply:
Since there’s an option in the UI to hide the `type` key, should we always be hiding it?